tensorflow/core/kernels/fused_batch_norm_op.h

```diff
@@ -43,6 +45,66 @@ struct InvVarianceToVariance {
   void operator()(const Eigen::GpuDevice& d, double epsilon, int sample_size,
                   int channels, T* variance);
 };
+
+
+// Functor used by FusedBatchNormGradOp to do the computations when is_training=False.
+// Both CPU and GPU will use this functor.
+template <typename Device, typename T>
+struct FusedBatchNormFreezeGrad {
```

Comment:
Should this be inside functor namespace? Note this test failure: tensorflow/core/kernels/fused_batch_norm_op.cc:645:7: error: 'FusedBatchNormFreezeGrad' is not a member of 'tensorflow::functor'

at https://ci.tensorflow.org/job/tensorflow-pull-requests-cpu-python3/6452/console

Reply:
Looks like this header is only included when GPU is enabled. I'll fix it.

Reply:
Ah this is more complicated than I thought. Looks like I need to change BUILD file somehow.